src/fs_scan/fs_scan.c

```diff
@@ -795,6 +801,35 @@ static int stat_entry(const char *path, const char *name, int parentfd,
     if (parentfd != -1) {
         if (fstatat(parentfd, name, inode, AT_SYMLINK_NOFOLLOW) == -1)
             return -errno;
+#ifdef _PANFS 
+        ssize_t vallentmp;
```

Comment:
I'd prefer this code is included to a subfunction here, to avoid overloading the code.